### ISSUE TITLE
php artisan broken in 6.0

### ISSUE THREAD

**Issue by newfield86** [laravel/lumen-framework#966]
- Lumen Version: 6.0
- PHP Version: 7.2.22

When I run `php artisan` in the new Lumen 6.0, I get the following error:

```
In RestartCommand.php line 40:

Too few arguments to function Illuminate\Queue\Console\RestartCommand::__construct(), 0 passed in /vendor/laravel/lumen-framework/src/Console/ConsoleServiceProvider.php on line 307 and exactly 1 expected
```

**Comment by driesvints** [comment]
Released v6.0.1 with fixes for this

**Comment by lloricode** [comment]
did not work
https://travis-ci.org/lloricode/lumen-dingo-boilerplate/jobs/595909439

**Comment by lloricode** [comment]
> Released v6.0.1 with fixes for this

did not work
https://travis-ci.org/lloricode/lumen-dingo-boilerplate/jobs/595909439

**Comment by Lorrhanl** [comment]
The version 6.0.1 did not solve for me. Only 6.2
